src/encounterApi/encounter.ts

```diff
@@ -0,0 +1,58 @@
+import { encounterApi } from '../config';
+import { getEncounterUserId } from '../helpers';
+import { parseEncounters, parseUser } from '../jsonValidation';
+
+export const createEncounterAsync = async (qrId: string) => {
+  const url = encounterApi.serverUrl + encounterApi.version + encounterApi.encounter.create;
+  const userId = await getEncounterUserId();
+
+  if (!userId) {
+    throw new Error('Error while loading encounter user id from storage.');
+  }
+
+  const body = JSON.stringify({ qr_code_id: qrId, user_id: userId });
+
+  const response = await fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body
+  });
+
+  const json = await response.json();
```

Comment:
could this be done only if (ok && status === 201) to avoid processing it in error cases?